docs/runbooks/KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert.md

```diff
@@ -0,0 +1,24 @@
+# KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert
+
+## Meaning
+The `KubevirtHyperconvergedClusterOperatorInstallationNotCompletedAlert` alerts means that the HypeConverged cluster operator (HCO) is running, but the `HyperConverged` custom resource does not exist.
+
+There are two reasons for that to happen: First, after installation of HCO, it is required to create the `HyperConverged` custom resource in order to complete the installation. The purpose of this alert is to guide the user to complete the installation by creating the `HyperConverged` custom resource. 
+
+The other reason for this alert is during uninstallation of HCO. In this case, the user should remove the `HyperConverged` CR before uninstalling the operator, and then again, the HCO is still running, while the `HyperConverged` custom resource was already removed, and so the alert will be triggered. In this case, the alert can be safely ignored. Completing the uninstallation process will cancel the alert.
+
+## Mitigation
+In case of installation, just create the HyperConverged custom resource to complete the installation; for example, to create the HyperConverged custom resource with default values, do:
```

Comment:
@nunnatsa Should you add `HyperConverged` here?

Reply:
@sradco - This is what described below - a simple example of deploying the HyperConverged custom resource.

Reply:
I meant to add ' before and after the word HyperConverged so it will be marked